app/controllers/image_controller.rb

```diff
@@ -268,7 +230,74 @@ def prev_image
     redirect_to_next_object(:prev, Image, params[:id].to_s)
   end
 
+  private
+
+  def image_exists
+    @image = find_or_goto_index(Image, params[:id].to_s)
+  end
+
+  def set_default_size
+    # Decide which size to display.
+    @default_size = @user ? @user.image_size : :medium
+    @size = params[:size].blank? ? @default_size : params[:size].to_sym
+
+    # Maybe make this size the default image size for this user.
+    unless @user && (@default_size != @size) && (params[:make_default] == "1")
+      return
+    end
```

Comment:
```suggestion
    return unless @user && 
                  (@default_size != @size) && 
                  (params[:make_default] == "1")
```
I like the guard clause (because it shows the non-normal path) instead of the old nested `if ... end`.
I'm merely suggesting avoiding making the guard clause itself nested. I think that also could be done with "`\`" line continuations.

Reply:
Done